src/BlazorUI/Bit.BlazorUI/Components/Inputs/Checkbox/BitCheckbox.scss

```
@@ -3,11 +3,16 @@
 .bit-chb {
     cursor: pointer;
     width: fit-content;
+    border-radius: $shp-border-radius;
     font-family: $tg-font-family;
     --bit-chb-ico-opa: 0;
     --bit-chb-box-clr-brd: #{$clr-brd-pri};
     --bit-chb-ico-clr: #{var(--bit-chb-clr-txt-sec)};
```

Stylelint (17.11.1)

[error] Missing var function for "--bit-chb-clr-txt-sec" (custom-property-no-missing-var-function)

(custom-property-no-missing-var-function)